src/controller/java/AndroidCallbacks.cpp

```diff
@@ -367,11 +363,11 @@ void ReportCallback::OnEventData(const app::EventHeader & aEventHeader, TLV::TLV
     jlong timestamp     = static_cast<jlong>(aEventHeader.mTimestamp.mValue);
 
     jobject value = DecodeEventValue(aEventHeader.mPath, readerForJavaObject, &err);
-    // If we don't know this event, just skip it.
-    VerifyOrReturn(err != CHIP_ERROR_IM_MALFORMED_EVENT_PATH_IB);
-    VerifyOrReturn(err == CHIP_NO_ERROR, ReportError(nullptr, eventPathObj, err));
-    VerifyOrReturn(!env->ExceptionCheck(), env->ExceptionDescribe(),
-                   ReportError(nullptr, eventPathObj, CHIP_JNI_ERROR_EXCEPTION_THROWN));
+    // If we don't know this event, just suppress it.
+    if (err != CHIP_NO_ERROR)
```

Comment:
`err = CHIP_NO_ERROR`. Comment should probably be updated to say *why* we suppress it. Also since we do not check just for MALFORMED_EVENT_PATH_IB we are saying here that any error we just let go through (and is this really correct?)

Reply:
If it is an error, I think it would be better to display the log for debug.